installers/common.sh

```diff
@@ -204,20 +206,37 @@ function _setup_synchrona_service() {
     RestartSec=1
     User=root
     ExecStartPre=-/usr/bin/sh -c "/usr/bin/echo 6 > /sys/class/gpio/export"
-    ExecStartPre=-/usr/bin/sh -c "/usr/bin/echo 12 > /sys/class/gpio/export"
-    ExecStartPre=-/usr/bin/sh -c "/usr/bin/echo 16 > /sys/class/gpio/export"
     ExecStartPre=-/usr/bin/dtoverlay -r rpi-ad9545-hmc7044
     ExecStartPre=-/usr/bin/dtoverlay /boot/overlays/rpi-ad9545-hmc7044.dtbo
     ExecStart=/usr/local/bin/uvicorn --app-dir=$tmp/app/python/synchrona  main:app --host 0.0.0.0 --port 8000
-    ExecStartPost=-/usr/bin/bash /etc/raspap/synchrona/ad-synchrona-14-leds.sh > /dev/null 2>&1 &
     ExecStopPost=-/usr/bin/sh -c "/usr/bin/echo 6 > /sys/class/gpio/unexport"
+
+    [Install]
+    WantedBy=multi-user.target
+    '
+
+    sudo -E bash -c 'cat > /etc/systemd/system/synchrona_leds.service <<- EOM
+    [Unit]
+    Description=Synchrona LEDs bash service
+    After=synchrona.service
+    PartOf=synchrona.service
+    StartLimitIntervalSec=0
+
+    [Service]
+    Type=simple
+    User=root
+    ExecStartPre=-/usr/bin/sh -c "/usr/bin/echo 12 > /sys/class/gpio/export"
+    ExecStartPre=-/usr/bin/sh -c "/usr/bin/echo 16 > /sys/class/gpio/export"
+    ExecStart=/usr/bin/bash /etc/raspap/synchrona/ad-synchrona-14-leds.sh > /dev/null 2>&1 &
```

Comment:
Do we still need to run the script in the background? Im fairly sure that systemd will already "daemonize" the script... I would also not redirect all the log to /dev/null as it might be useful to run `journalctl -u synchrona_leds` to look for possible issues